### DISCUSSION TITLE
App ignores language setting, reverts to System

### DISCUSSION THREAD

**Discussion by elakapmain** [libre-tube/LibreTube#6676] [General]
### Steps to reproduce

Strange, oddly specific bug. Not sure if it's just my device.

1. Set your phone language to anything other than English 
2. In the LibreTube app, set the app language to English from System
3. Play any video (video must load and play)
2. Minimize it
4. Change tabs via navigation bar

### Expected behavior

The app language should not change.

### Actual behavior

The app language changes.

However, the settings page is not affected. Only certain screens are.

### LibreTube version

0.25.2

### Android version

Android 15

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
An app restart should take care of such issues usually.

**Reply by elakapmain** [reply]
An app restart does take care of the issue, but it is constantly reproducible on my end.

(My phone language is in Japanese)
https://github.com/user-attachments/assets/5c980675-d376-4afc-9d27-634f7fa31ebf

**Reply by elakapmain** [reply]
Yes. This is a problem with the app. I'm not sure why this isn't reproducible elsewhere though.

**Reply by elakapmain** [reply]
So you're experiencing this issue on Android 12? I'm having this on 15.

**Reply by elakapmain** [reply]
Oh great. That means it's reproducible then. A video would be helpful, but it's good to know it's not just me.
